tests/test_optim.py

```diff
@@ -66,18 +66,18 @@ def test_minimize(tensorlib, precision, optimizer, do_grad, do_stitch):
             'no_grad-scipy-tensorflow-32b': [1.0, 1.0],
             'no_grad-scipy-jax-32b': [1.0, 1.0],
             # no grad, scipy, 64b
-            'no_grad-scipy-numpy-64b': [0.49998815367220306, 0.9999696999038924],
-            'no_grad-scipy-pytorch-64b': [0.49998815367220306, 0.9999696999038924],
-            'no_grad-scipy-tensorflow-64b': [0.49998865164653106, 0.9999696533705097],
-            'no_grad-scipy-jax-64b': [0.4999880886490433, 0.9999696971774877],
+            'no_grad-scipy-numpy-64b': [0.4999985, 1.00000122],
+            'no_grad-scipy-pytorch-64b': [0.4999985, 1.00000122],
+            'no_grad-scipy-tensorflow-64b': [0.49999869, 1.00000121],
+            'no_grad-scipy-jax-64b': [0.49999894, 1.00000114],
```

Comment:
@lukasheinrich @kratsg While these changes are relatively small (mostly on the O(e-6) to O(e-7) range) I'm not sure if it makes sense to me that we're still seeing these, or the JAX change that is quite a bit larger.

Reply:
these are ok, and self-consistent and close to [0.5,1.0]